spyder/plugins/variableexplorer/widgets/objectexplorer/utils.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright (c) 2016 Pepijn Kenter.
+# Copyright (c) 2019- Spyder Project Contributors
+#
+# Components of objectbrowser originally distributed under
+# the MIT (Expat) license.
+# Licensed under the terms of the MIT License; see NOTICE.txt in the Spyder
+# root directory for details
+# -----------------------------------------------------------------------------
+
+# Standard library imports
+import logging
+
+
+def logging_basic_config(level='INFO'):
+    """
+    Setup basic config logging. Useful for debugging to
+    quickly setup a useful logger.
+    """
+    fmt = '%(filename)25s:%(lineno)-4d : %(levelname)-7s: %(message)s'
+    logging.basicConfig(level=level, format=fmt)
+
+
+def check_class(obj, target_class, allow_none=False):
```

Comment:
This function doesn't seem to be used either in this PR. If that's the case, please remove it.